cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json

```diff
@@ -0,0 +1,460 @@
+[
+  {
+    "abstract": {
+      "description": "<p>This file describes which luminosity sections in which runs are considered good and should be processed.</p> <p>This list covers heavy-ion data taking in 2010, between run numbers 150436 and 152957.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "CMS Collaboration",
+      "recid": "450"
+    },
+    "collections": [
+      "CMS-Validated-Runs",
+      "CMS-Validation-Utilities"
+    ],
+    "date_created": [
+      "2010"
+    ],
+    "date_published": "2020",
+    "distribution": {
+      "formats": [
+        "txt"
+      ]
+    },
+    "experiment": "CMS",
+    "publisher": "CERN Open Data Portal",
+    "recid": "14202",
+    "run_period": [
+      "HIRun2010"
+    ],
+    "title": "CMS list of validated runs Cert_150436-152957_HI7TeV_StreamExpress_Collisions10_JSON_v2.txt",
```

Comment:
The title field says `Cert_150436-152957` but the files themselves are called `Cert_150436-151238`, which one is correct?

Reply:
Thanks for pointing that out! The files are wrong, the title is correct. I'll upload the new ones right now.

Reply:
The two correct files are now in https://cernbox.cern.ch/index.php/s/D236HWZGuMOnSZB. This affects both records having `Cert_150436-152957`.

Reply:
Thanks, all updated, repushing inside the same PR, you'll be able to verify everything later on DEV...